recipes/pyglfw/run_tests.py

```diff
@@ -0,0 +1,13 @@
+import os
+import sys
+
+# Ensure the GLFW library can be found.
+CONDA_PREFIX = os.environ['CONDA_PREFIX']
+
+if sys.platform == 'linux':
+    os.environ['LD_LIBRARY_PATH'] = CONDA_PREFIX + os.path.sep + 'lib'
+elif sys.platform == 'darwin':
+    os.environ['DYLD_LIBRARY_PATH'] = CONDA_PREFIX + os.path.sep + 'lib'
```

Comment:
if you need to do this, something is wrong

Reply:
Hm ok? I didn't find another way to fix this, though…

Reply:
Would that be a problem with the `glfw` package, then?

Reply:
@scopatz  Wonder if this could be a solution?
https://github.com/conda/conda/issues/1533#issuecomment-132737669

Reply:
`pyGLFW` will also check the `PYGLFW_LIBRARY` path variable to locate the `glfw` library. Would it be possible to modify the `glfw` package to set this variable automatically once the respective `conda` environment is being activated?

Reply:
Seems to be an ongoing problem with numerous libs:
https://github.com/ContinuumIO/anaconda-issues/issues/1716

Reply:
@scopatz This needs to be fixed upstream, no?
https://github.com/FlorianRhiem/pyGLFW/blob/master/glfw/library.py#L123-L144

An alternative could be to [set `PYGLFW_LIBRARY`](https://github.com/FlorianRhiem/pyGLFW/blob/master/glfw/library.py#L147-L149), but I don't know how one could do that automatically one the environment gets activated?

Reply:
I've included a patch.